onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -389,6 +389,7 @@ def _where_input_wrangler(
     "tan": core_ops.aten_tan,
     "tanh": core_ops.aten_tanh,
     "topk": core_ops.aten_topk,
+    "trunc": core_ops.aten_trunc,
```

Comment:
I wonder if there is anything we can do to improve the clarity on actual test cases. Seems hard to check without running it.

Reply:
Any suggestions? What kind of information will be useful?